node/src/components/chainspec_loader/config.rs

```diff
@@ -22,7 +22,11 @@ const DEFAULT_CHAIN_NAME: &str = "casper-devnet";
 const DEFAULT_ACCOUNTS_CSV_PATH: &str = "accounts.csv";
 const DEFAULT_VALIDATOR_SLOTS: u32 = 5;
 const DEFAULT_AUCTION_DELAY: u64 = 3;
-const DEFAULT_LOCKED_FUNDS_PERIOD: EraId = 15;
+
+/// Default lock-in period of 90 days
+/// <https://www.wolframalpha.com/input/?i=convert+90+days+to+milliseconds>
+const DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS: u64 = 7776000000; // 90 days
```

Comment:
As above, you could write it as a product?

Reply:
Donezo.